app/src/main/res/layout/main.xml

```diff
@@ -1,153 +1,150 @@
 <?xml version="1.0" encoding="utf-8"?>
 <android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:app="http://schemas.android.com/apk/res-auto"
-  android:id="@+id/snackbar_layout"
-  android:layout_width="match_parent"
-  android:layout_height="match_parent"
-  android:fitsSystemWindows="false"
-  android:orientation="vertical">
-
-
-  <android.support.v4.widget.DrawerLayout
-    android:id="@+id/drawer_layout"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/snackbar_layout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:fitsSystemWindows="true">
-
-    <android.support.design.internal.ScrimInsetsFrameLayout
-      android:id="@+id/content_frame"
-      android:layout_width="match_parent"
-      android:layout_height="match_parent"
-      android:fitsSystemWindows="true"
-      android:orientation="vertical" />
-
-    <RelativeLayout
-      android:id="@+id/MainLayout"
-      android:layout_width="fill_parent"
-      android:layout_height="fill_parent"
-      android:orientation="vertical">
-
-      <RelativeLayout
-        android:id="@+id/FullscreenButtonLayout"
-        android:layout_width="fill_parent"
-        android:layout_height="fill_parent">
+    android:fitsSystemWindows="false"
+    android:orientation="vertical">
+
+
+    <android.support.v4.widget.DrawerLayout
```

Comment:
Should be 2 space indentation in multiple places

Reply:
@mhutti1 Can you please check now?